plugins/discourse-rewind/spec/services/fetch_report_spec.rb

```diff
@@ -96,7 +93,11 @@
     end
 
     context "when reports are not cached" do
-      before { Discourse.redis.del("rewind:#{current_user.username}:2021") }
+      before do
+        Discourse.redis.del(
+          DiscourseRewind::FetchReportsHelper.cache_key(current_user.username, 2021),
+        )
+      end
```

Comment:
I think we have `Discourse.redis.flushdb` in rails_helper's `config.after :each do |example|`. Perhaps we can omit this

Reply:
Yah we do, weird that we have a bunch of other specs calling `Discourse.redis.flushdb` manually  too. Fixing for rewind

Reply:
Oh wait for this specific one we need it ,because we are setting the cache  in the `before` for the  whole spec...but maybe i can just rearrange stuff